src/renderer/Renderer.ts

```diff
@@ -105,15 +110,49 @@ export class Renderer {
                 positions: o.vertices,
                 normals: o.normals,
                 colors: o.colors,
-                indices: o.indices
-            })
+                indices: o.indices,
+                numLights: this.lights.length,
+                maxLights: this.maxLights,
```

Comment:
tiny nit: we can take out `maxLights` here since we pass it in along with `regl` earlier so this prop doesn't get used